systemtests/tests/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@
 #   02110-1301, USA.
 
 add_subdirectory(acl)
-add_subdirectory(ai-consolidate-ignore-duplicate-job)
+add_subdirectory(always-incremental-consolidate)
```

Comment:
Maybe you want your last commit be amended to adjust the description systemtest: renamed ai-consolidate-ignore-duplicate-job to always-incremental-consolidate

Reply:
sure!